cdip_connector/core/logconfig.py

```diff
@@ -28,6 +28,15 @@
         "PIL.Image": {
             "level": "INFO",
         },
+        'httpx': {
+            'handlers': ['console'],
+            'level': 'INFO',
+        },
+        'httpcore': {
+            'handlers': ['console'],
+            'level': 'INFO',
+        },
+
```

Comment:
httpx isn't used (yet) in this connector code, but it might be in integration implementations (ex. Savannah Tracking). So I've added filters here to be sure our production logs won't become too noisy.